### PR TITLE
Clarify US Emissions Gap Clearer

### LINKED ISSUE #129: Clarify Missing Emissions Data for 2019-2022 in Area Chart
There's a gap between 2018 and 2022 in our area chart:
Screen Shot 2022-07-19 at 7 56 34 PM

Option 1: find state totals for the missing years and fill them i9n

Option 2: show greyed out estimate section for the missing years

### PR DESCRIPTION
**Pull request by hkier**
## Overview

This PR installs the code which fixes the problem where there are years with no data and were previously excluded from the graph.  This created a situation where the x-axis of the graph was not contiguous.  The years with missing data are now included on the graph but are shown in grey to indicate the data is unavailable.

Closes #129

Screenshot 2022-07-30 at 15-39-16 What does it take to decarbonize your state

### Notes

None

## Testing Instructions
This PR is tested by visually viewing the graph for several different states to verify the graph is created as expected and shown in the example about.

### CODE REVIEW

**Comment by vkoves** [comment]
@derekeder - would love your thoughts, but I'd suggest two visual tweaks - one is to make the grey line dashed (but the green line isn't so 🤷🏻‍♂️ ) and the other would be to change "Missing Data" to just "No Data". 

Alternatively we could remove the label text and instead put some text underneath that says "Grey area indicates missing data due to lag in data processing".

Also, @hkier is this properly future-proof? Like would it work in 2025 if we didn't update the data and show the gap expanding (I tried to test this by manipulating my clock, but that broke HTTPS and stopped me loading the page)

File: src/components/simpleareachart.js [hunk 1]
```diff
@@ -19,28 +19,37 @@ export default function SimpleAreaChart ({ emissionsData, title }) {
...
   const currYear = new Date().getFullYear()
   const yearsLeft = 2050 - currYear
-  const reduceFrom
-    = annualHistoricEmissions[annualHistoricEmissions.length - 1][EmissionsDataKey]
+  const reduceFrom = annualHistoricEmissions[annualHistoricEmissions.length - 1][EmissionsDataKey]
+  const lastYear = annualHistoricEmissions.slice(-1)[0].year
   var projection = []
+  var missing = []
+
+  for (let step = 0; step < (currYear - lastYear - 1); step++) {
```
Comment (vkoves) [inline src/components/simpleareachart.js:40]: Can you document this code so folks know why it's doing what it does?

**Comment by hkier** [comment]
@vkoves actually the screen shot is slightly out of date.  I too did not like the verbiage and it has been changed to "Data Unavailable"

**Comment by hkier** [comment]
Yes, the code is future proofed as I use calculations based on the last year of available data and the current year.  There should not be a problem as the clock advances.

File: src/components/simpleareachart.js (hunk 1 above)
Comment (hkier) [reply src/components/simpleareachart.js:48]: Done.

**Comment by vkoves** [comment]
@hkier - what do you think of my idea with no label and an asterisk below? It is cleaner (no more three lines of text right near each other) but might confuse people so I'm torn:

Screenshot from 2022-08-02 19-42-59

If we keep the text though we should make sure it's centered in the grey area - right now it's weighted towards the right side

**Comment by vkoves** [comment]
In the call tonight we settled on this:

Screenshot from 2022-08-02 21-09-58

Changes are:

1. Grey area has dashed line (also thickened all the borders, but that feels optional)
2. Removed label for grey area
3. Changed label styling for other areas, moving to horizontal text at larger size, centered
4. Added note about grey area above the graph, in 14px size text (to make it smaller than the emissions explainer)

**Comment by vkoves** [comment]
Actually mobile looks hard to read so I think we need to fix this:

Screenshot from 2022-08-30 20-59-59

**Comment by vkoves** [comment]
@derekeder - since I made some minor changes fixing mobile (bringing us halfway towards the improvements in #152), could you review this? Still pretty simple code

**Review by derekeder — APPROVED** [review]

**Pull request merged**